src/UI/components/about/aboutTeam.test.tsx

```diff
@@ -12,4 +14,14 @@ describe(AboutTeam.name, () => {
       screen.getByTestId('teamListContainer').children.length;
     expect(numRenderedMembers == teamMembers.length);
   });
+
+  it('Opens a div when avatar is clicked', async () => {
+    await act(async () => {
+      renderWithUser(<AboutTeam />);
+    });
+    const openBox = screen.getByTestId('teamListbox');
+
+    fireEvent.click(openBox);
+    expect(openBox).toBeInTheDocument();
```

Comment:
I don't think this is testing anything, the `openBox` is in the document already as it's what is clicked in line 24

Reply:
do i remove the test or rewrite it for when an icon is clicked a div opens